src/Streamly/Internal/Data/Fold.hs

```diff
@@ -27,21 +28,21 @@ module Streamly.Internal.Data.Fold
     , mkFoldM
     , mkFoldM_
 
-    -- * Generators
+    -- * Folds
```

Comment:
This is the `Fold` module. Don't think this heading is necessary.

Reply:
The module is divided into `Folds` and `Fold Combinators`. This section needs to have a name, since it consists of folds that is an appropriate name for it.